editor/src/components/canvas/canvas-strategies/flex-resize-basic-strategy.tsx

```diff
@@ -144,24 +149,36 @@ export function flexResizeBasicStrategy(
 
           const resizeCommands: Array<AdjustCssLengthProperty> = []
 
-          const newWidth = makeNewDimension(
+          const parentWidth = elementParentBounds?.width
+          const parentHeight = elementParentBounds?.height
+
+          let newWidth = makeNewDimension(
             originalBounds.width,
             resizedBounds.width,
             dimensions?.width,
           )
-          if (dimensions?.width != null || originalBounds.width !== newWidth) {
-            // it moves horizontally
-            resizeCommands.push(makeResizeCommand('width', elementParentBounds?.width, newWidth))
-          }
-
-          const newHeight = makeNewDimension(
+          let newHeight = makeNewDimension(
             originalBounds.height,
             resizedBounds.height,
             dimensions?.height,
           )
+          if (lockedAspectRatio) {
+            if (newWidth) {
```

Comment:
Please can we avoid using truthiness, and instead explicitly perform null or zero checks

Reply:
crap, overlooked this one. fixing